tests/test_metrics.py

```diff
@@ -66,8 +64,8 @@ def test_provable_vs_adjusted(self):
         x1 = tf.random.normal((n, 10), mean=0.0, stddev=0.1)
         x2 = tf.random.normal((n, 10), mean=1.0, stddev=0.1)
         for lip_cst in [1.0, 0.5, 2.0]:
-            bar = BinaryAdjustedRobustness(lip_cst)
-            bpr = BinaryProvableRobustness(lip_cst)
+            bpr = ProvableAvgRobustness(lip_cst, disjoint, negative_robustness=False)
+            bar = ProvableAvgRobustness(lip_cst, disjoint, negative_robustness=True)
```

Comment:
I found names `bpr` and `bar` confusing because they are not binary metrics anymore...

Reply:
the metrics are now handling both binary and multiclass now :-)